jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BottomNavigationTests.java

```diff
@@ -17,6 +17,15 @@
 
 public class BottomNavigationTests extends TestsInit {
 
+    private final String recents = "Recents";
```

Comment:
you can use enum and process set of values into checking of all available options.